setup.py

```diff
@@ -7,16 +7,7 @@
     version='0.1.0',
     packages=find_packages(),
     install_requires=[
-        'python-dotenv',
```

Comment:
this being dropped is my best guess of where the test failure is coming from

Reply:
Just re-added, thx.